### PR TITLE
Feat/19how we work section

### PR DESCRIPTION
**Pull request by LULIU-9**
### Descriptions

- Link to UI design (if possible):
https://www.figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=104%3A319&t=CZe2ZOr7ajfzETSg-4

### Provide screenshots or videos that the function is working.
Screen Shot 2023-02-22 at 8 37 19 pm
Screen Shot 2023-02-22 at 8 39 43 pm

### Provide screenshots of no warning, or error in the console.

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Comment by Cameron933** [comment]
1. Provide the Figma url instead of screenshot.
2. assign yourself to your PR
image

File: styles/HowWeWork.module.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.how_we_work {
+  display: grid;
+  grid-template-columns: 69% 31%;
```
Comment (Cameron933) [inline styles/HowWeWork.module.css:3]: Please refer to the CSS of Contact section to keep consistency
image

File: styles/HowWeWork.module.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  height: 100px;
+  color: var(--icon-default);
+}
+.how_we_work__content {
+  border-left: 1px solid var(--border-default);
+  padding-left: 48px;
+}
+.how_we_work__content_text {
+  height: 464px;
+  font-weight: 400;
```
Comment (Cameron933) [inline styles/HowWeWork.module.css:30]: This should be defined in font styles, not here.

File: __tests__/index.test.jsx [hunk 3]
```diff
@@ -55,6 +55,37 @@ describe("Home", () => {
...
+  });
+
+  it("should render how we work content in sucess", () => {
+    const howWeWorkContent = screen.getByText(
+      "Our team of experts is dedicated to delivering real results through creativity, strategic thinking, and technical expertise. Whether it‘s developing a new marketing campaign, designing a website, or creating a mobile app, we are always pushing the boundaries of what’s possible to help our clients stay ahead of the curve."
+    );
+    expect(howWeWorkContent).toBeInTheDocument();
+  });
+
+  it("should render how we work button in sucess", () => {
```
Comment (Cameron933) [inline __tests__/index.test.jsx:84]: it should render the button in how we work section in success.

File: __tests__/index.test.jsx [hunk 4]
```diff
@@ -55,6 +55,37 @@ describe("Home", () => {
...
+    const howWeWorkButton = screen.getAllByRole("button", { name: /OUR SERVICES/i });
+    expect(howWeWorkButton[0]).toBeInTheDocument();
```
Comment (Cameron933) [inline __tests__/index.test.jsx:86]: Why [0]? This section only has one button

**Review by Cameron933 — APPROVED** [review]
LGTM

**Review by HarryuCATGREY — APPROVED** [review]
LGTM

**Pull request merged**